task/04-arrays-tasks.js

```diff
@@ -200,7 +200,7 @@ function getTail(arr, n) {
  *    +'30,31,32,33,34'
  */
 function toCsvText(arr) {
-   throw new Error('Not implemented');
+    return arr.map(row => row.join(',')).join("\n");
```

Comment:
Solution can be simplified to this:

```suggestion
    return arr.join("\n");
```